### PR TITLE
Update VMHostAssignableDevice command set - WS22

### PR DESCRIPTION
**Pull request by Xelu86**
# PR Summary

Populated information for the VMHostAssignableDevice command set for Windows Server 2022.

## PR Checklist

- [x] **Descriptive Title:** This PR's title is a synopsis of the changes it proposes.
- [x] **Summary:** This PR's summary describes the scope and intent of the change.
- [x] **Contributor's Guide:** I have read the [contributors guide][contrib].
- [x] **Style:** This PR adheres to the [style guide][style].

[contrib]: https://learn.microsoft.com/powershell/scripting/community/contributing/overview
[style]: https://learn.microsoft.com/powershell/scripting/community/contributing/powershell-style-guide

### CODE REVIEW

**Comment by officedocspr4** [comment]
Thanks for submitting your pull request. To improve the quality of your content and make sure all errors are resolved, we recommend that you use the [PubOps publishing process](https://review.docs.microsoft.com/office-authoring-guide/pubops-publishing-process?branch=main).

1. Add the **Sign off** label when you think your PR is ready to merge. A PR reviewer either merges it, or adds the **Back to Submitter** label with a comment about [what needs to be fixed](https://review.docs.microsoft.com/office-authoring-guide/pr-criteria?branch=main) and sends it back to you.

3. If you need to update your content per the PR reviewer feedback, add the **Sign off** label again.

4. Your content is published live during one of the next publishing cycles, which are weekdays at 10:30 AM Pacific Time and 3:30 PM Pacific Time.

If you have a question, post it on the [Ask an Admin](https://teams.microsoft.com/l/channel/19%3aea9a561f603c4fbb9cf3931a6849eae4%40thread.skype/Ask%2520an%2520Admin?groupId=7079ffbf-1420-4ee2-9ddc-a2c351337320&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) support channel.

File: docset/winserver2022-ps/hyper-v/Add-VMHostAssignableDevice.md [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ComputerName
+
+Specifies one or more Hyper-V hosts on the virtual network adapters are to be retrieved. NetBIOS
```
Comment (jackschefer-msft) [inline docset/winserver2022-ps/hyper-v/Add-VMHostAssignableDevice.md:78]: What does "on the virtual network adapters are to be retrieved" mean?

File: docset/winserver2022-ps/hyper-v/Dismount-VMHostAssignableDevice.md [hunk 2]
```diff
@@ -0,0 +1,218 @@
+---
+description: Dismounts an assigned device from a virtual machine host.
```
Comment (jackschefer-msft) [inline docset/winserver2022-ps/hyper-v/Dismount-VMHostAssignableDevice.md:2]: Suggestion: "Dismounts an **assignable** device..."

The device shouldn't be assigned anywhere at this point, these commandlets are for managing the devices on the host before assigning (or after assigning)

File: docset/winserver2022-ps/hyper-v/Dismount-VMHostAssignableDevice.md [hunk 3]
```diff
@@ -0,0 +1,218 @@
...
+## EXAMPLES
+
+### Example 1
+
+```powershell
+Dismount-VMHostAssignableDevice -InstancePath "PCIROOT(0)#PCI(1D02)#PCI(0000)"
+```
+
+This example dismounts the device with the specified instance path from the host, making it
+available for reassignment to another virtual machine.
```
Comment (jackschefer-msft) [inline docset/winserver2022-ps/hyper-v/Dismount-VMHostAssignableDevice.md:39]: Suggestion: "... making it available for **assignment to a virtual** machine."

The physical VM hosting machine is not a VM, so consumers might find it confusing to talk about reassignment

File: docset/winserver2022-ps/hyper-v/Dismount-VMHostAssignableDevice.md [hunk 4]
```diff
@@ -0,0 +1,218 @@
...
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Force
+
+Forces the command to run without asking for user confirmation.
```
Comment (jackschefer-msft) [inline docset/winserver2022-ps/hyper-v/Dismount-VMHostAssignableDevice.md:99]: `-Force` also overrides some security checks in the platform, see also: https://learn.microsoft.com/en-us/windows-server/virtualization/hyper-v/deploy/deploying-graphics-devices-using-dda#install-the-partitioning-driver-optional

> Install the partitioning driver (optional)
DDA gives hardware vendors the ability to provide a security mitigation driver with their devices. This driver isn't the same as the device driver installed in the guest VM. It's up to the hardware vendor's discretion to provide this driver. But if they do provide a driver, install it before dismounting the device from the host partition. Reach out to the hardware vendor to see if they have a mitigation driver.

If no partitioning driver is provided, during dismount you must use the -Force option to bypass the security warning. For more information about the security implications, see [Plan for Deploying Devices using Discrete Device Assignment](https://learn.microsoft.com/en-us/windows-server/virtualization/hyper-v/plan/plan-for-deploying-devices-using-discrete-device-assignment).

File: docset/winserver2022-ps/hyper-v/Get-VMHostAssignableDevice.md [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -ComputerName
+
+Specifies one or more Hyper-V hosts on the virtual network adapters are to be retrieved. NetBIOS
```
Comment (jackschefer-msft) [inline docset/winserver2022-ps/hyper-v/Get-VMHostAssignableDevice.md:72]: What does "on the virtual network adapters are to be retrieved." mean here?

**Pull request merged**